### DISCUSSION TITLE
OAuth PKCE flow for ASP.NET Core with Swagger

### DISCUSSION THREAD

**Discussion by giscus** [CamiloTerevinto/Blog#3] [Announcements]
# OAuth PKCE flow for ASP.NET Core with Swagger

https://www.camiloterevinto.com/post/oauth-pkce-flow-for-asp-net-core-with-swagger

**Comment by odyyudah** [comment]
please share GitHub samples repository. thank you!

**Reply by CamiloTerevinto** [reply]
Hey @odyyudah - here’s the link: https://github.com/CamiloTerevinto/Blog/tree/main/Samples/OAuth%20PKCE%20flow%20for%20Swagger%20in%20ASP.NET%20Core

**Reply by odyyudah** [reply]
Thank you!
